source/databricks/calculation_engine/setup.py

```diff
@@ -46,6 +46,7 @@
         "console_scripts": [
             "start_calculator = package.calculator_job:start",
             "migrate_data_lake = package.datamigration.migration:migrate_data_lake",
+            "optimise_tables = package.optimise_job.optimisation:optimise_tables",
```

Comment:
nit: It would be nice to read this as:
```suggestion
            "optimize_delta_tables = package.delta_optimization:optimize",
```

Reply:
That would require we don't have a sub-folder for the optimize job, and make it unlike the datamigration which also has it's own sub-folder